### PR TITLE
Add project reports to the rocket pages for uxo and vidar3

### PR DESCRIPTION
**Pull request by akmorrison**
This puts the final submitted versions of the project report pdfs for
2017 and 2018 on the team website, and includes a link to them from
the rocket pages for UXO and VIDARIII. We've been discussing how much
we value sharing information about our rockets, and this PR is an
extension of those discussions.

I put the reports on their respective rocket pages because it was the
minimum effort required. I considered making a new page to host all of
the reports from all time, but that seemed like more effort than it
was worth. I also only added the two most recent ones, if anyone feels
that all the old reports should be put up, I can do that.

### CODE REVIEW

File: vidar3.html [hunk 1]
```diff
@@ -103,6 +103,10 @@ <h1 class="content-title">NOSECONE</h1>
 					<p class="content-text">
 						The nosecone is an elliptical shape, with a length of 8". The elliptical shape was chosen because the rocket is expected to remain subsonic. It is machined out of pine wood, and coated with epoxy.
 					</p>
+					<h1 class="content-title">PROJECT REPORT</h1>
+					<p class="content-text">
+					  A complete report of this project, including descriptions of onboard and ground support systems, engineering drawings of all rocket components, and a complete set of assembly and launch procedures for VIDAR III can be downloaded <a href="pdfs/201_project_report.pdf">here</a>.
```
Comment (jacobdeery) [inline vidar3.html]: Change to Vidar III
Link should be `pdfs/2017_project_report.pdf`

Comment (akmorrison) [reply vidar3.html]: Whoops, my bad, fixed

Comment (jacobdeery) [reply vidar3.html]: This was two comments. You fixed the link, but "VIDAR III" should be "Vidar III".

Comment (akmorrison) [reply vidar3.html]: Fuck. I should read better

**Review by jacobdeery — APPROVED** [review]

**Pull request merged**